torntools/scripts/content/global/ttGlobal.css

```diff
@@ -1097,3 +1097,64 @@ div[role="main"].tt-modified {
 .tt-mobile .tt-upkeep span:not([class*="svgIconWrap"]) {
 	color: #85b200 !important;
 }
+
+div#tt-loot {
+	font-size: 14px;
+	background-color: #f2f2f2;
+	border-radius: 0 5px 5px 0;
+}
+
+.dark-mode div#tt-loot {
+	background-color: #333;
+}
+
+div#tt-loot > div:first-child {
+	padding-top: 5px;
+}
+
+div#tt-loot > div:last-child {
+	padding-bottom: 5px;
+}
+
+div#tt-loot > div {
+	padding-bottom: 5px;
+	width: fit-content;
+}
+
+div#tt-loot > div > div {
```

Comment:
Using classes makes it much more easier to understand where you apply styles to.